package.json

```
@@ -38,13 +38,13 @@
     "mocha": "^10.0.0",
     "ts-mocha": "^10.0.0",
     "ts-node": "^10.9.1",
+    "@artus-cli/artus-cli": "latest",
     "tsconfig-paths": "^4.1.1",
-    "tslib": "^2.4.0",
     "typescript": "^4.8.2"
   },
   "repository": {
     "type": "git",
-    "url": ""
+    "url": "https://github.com/artus-cli/plugin-version"
```

```suggestion
    "url": "git@github.com:artus-cli/plugin-version.git"
```

这里要 git 开头的那个地址，不然会有问题